fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/GenericDataServiceImpl.java

```diff
@@ -88,7 +88,7 @@ public GenericResultsetData fillGenericResultSet(final String sql) {
                 resultsetDataRows.add(resultsetDataRow);
             }
 
-            return new GenericResultsetData(columnHeaders, resultsetDataRows);
+            return new GenericResultsetData(columnHeaders, resultsetDataRows, 0, 0);
```

Comment:
I mean 0, 0 makes sense here? I don't think so.
The total size of the result should be the size of the resultsetDataRows, isn't it? Same for the recordsPerPage since this is not a paginated API.
Let's try to be consistent with the data we're trying to represent.